examples/eth_bots/config.py

```diff
@@ -58,17 +58,26 @@
     ),
 ]
 
+# TODO: Clean up variables that are only used by evm_bots & apeworx
 environment_config = EnvironmentConfig(
     alchemy=False,
-    artifacts_url="http://localhost:80",
     delete_previous_logs=False,
     devnet=True,
     halt_on_errors=True,
     log_filename="agent0-bots",
     log_level=logging.INFO,
     log_stdout=True,
-    rpc_url="http://localhost:8545",
     random_seed=1234,
-    username="changeme",
+    # TODO: should just use
+    # hostname = http://localhost
+    # artifacts_port = 80
+    # rpc_port = 8545
+    # user_registry_port = 5002
+    artifacts_url="http://localhost:80",
+    rpc_url="http://localhost:8545",
     username_register_url="http://localhost:5002",
+    # artifacts_url="http://ec2-3-13-94-236.us-east-2.compute.amazonaws.com:80",
```

Comment:
nit, I'd say lets leave out the actual url endpoint from the repo. I guess now it's committed though it's in the history forever...

Reply:
I also would have said this. hmmm.  we _can_ overwrite this.  the risk is pretty low in this case so we're lucky.  remember these are totally unsecure ports.

Reply:
no it'll be gone on squash. I'll remove it.